### ISSUE TITLE
`bundle viz` deprecation path

### ISSUE THREAD

**Issue by deivid-rodriguez** [bundler/bundler#7041]
I'm reviewing the `bundle viz` deprecation path, and it needs quite a bit of work.

* Current deprecation message says 

    > [DEPRECATED] The `viz` command has been moved to the `bundle-viz` gem, see https://github.com/bundler/bundler-viz.

    but that repo doesn't exist. We should either move https://github.com/kerrizor/bundler-dependency_graph to the bundler organization, or change the message to point to the alternative repo. The former sounds better.

* The deprecation message does not give any information about how to migrate to the plugin. We should change the message to something like:

    > [DEPRECATED] The `viz` command has been moved to a bundler plugin, see https://github.com/bundler/bundler-viz. In order to use the new plugin, run `bundle plugin install bundler-dependency_graph`. After that, you can run `bundle graph` instead of `bundle viz` to get a dependency graph for your application.

* The status of the new repo should be reviewed. Currently the plugin seems to work, although it prints the message

    > Currently unimplented; use `bundle viz` to generate your graphs for now.

    That, plus the list of issues in the repo, makes it sound like it's still a WIP.

How do we proceed?

**Comment by colby-swandale** [comment]
The idea is just copy the implementation of the `bundler viz` command into a plugin.

**Comment by colby-swandale** [comment]
We need to copy the implementation of the `bundle viz` command into a plugin and update the error message with instructions to use the new plugin.

**Comment by deivid-rodriguez** [comment]
Yeah, that's already done at https://github.com/kerrizor/bundler-dependency_graph. My question is whether we should use that repo, or move it under the `bundler/` umbrella.

**Comment by fatkodima** [comment]
@deivid-rodriguez @colby-swandale 

I've made up-to-date extraction here https://github.com/fatkodima/bundler-visualize
Let me know, what you think.
